apps/prairielearn/src/sprocs/sync_course_instances.sql

```diff
@@ -152,10 +152,12 @@ BEGIN
         display_timezone = COALESCE(src.data->>'display_timezone', c.display_timezone),
         hide_in_enroll_page = (src.data->>'hide_in_enroll_page')::boolean,
         json_comment = (src.data->>'comment')::jsonb,
+        modern_publishing = (src.data->>'modern_publishing')::boolean,
         self_enrollment_enabled = (src.data->>'self_enrollment_enabled')::boolean,
         self_enrollment_enabled_before_date = input_date(src.data->>'self_enrollment_enabled_before_date', COALESCE(src.data->>'display_timezone', c.display_timezone)),
         self_enrollment_use_enrollment_code = (src.data->>'self_enrollment_use_enrollment_code')::boolean,
-        self_enrollment_restrict_to_institution = (src.data->>'self_enrollment_restrict_to_institution')::boolean,
+        publishing_start_date = input_date(src.data->>'publishing_start_date', COALESCE(src.data->>'display_timezone', c.display_timezone)),
+        publishing_end_date = input_date(src.data->>'publishing_end_date', COALESCE(src.data->>'display_timezone', c.display_timezone)),
```

Comment:
Nit, we were sorta-kinda maintaining alphabetical order before, should we continue to?

Reply:
Fixed in 4ebd7b33a3f4ea5bcbed06c9ac48c9184f318b52. I also rearranged the params passed via typescript.